hypertrace-gradle-docker-plugin/src/main/java/org/hypertrace/gradle/docker/DockerPlugin.java

```diff
@@ -109,19 +112,23 @@ private boolean isReleaseVersion(Project project) {
   }
 
   private String getBranchTag() {
-    // Use the value of CIRCLE_BRANCH environment variable if defined (that is, the branch name used
-    // to build in CI), otherwise for local builds use 'test'
-    return getEnvironmentVariable("CIRCLE_BRANCH")
-        .map(String::trim)
-        .map(branch -> branch.replaceAll("[^A-Za-z0-9\\.\\_\\-]", ""))
-        .filter(branch -> !branch.isEmpty())
-        .orElse("test");
+      // Use the value of CIRCLE_BRANCH/ GHA_BRANCH environment variable if defined (that is, the branch name used
+      // to build in CI), otherwise for local builds use 'test'
+      return getEnvironmentVariable("CIRCLE_BRANCH")
+          .map(String::trim)
+          .map(branch -> branch.replaceAll("[^A-Za-z0-9\\.\\_\\-]", ""))
+          .filter(branch -> !branch.isEmpty())
+          .orElse(getEnvironmentVariable("GITHUB_REF")
+                  .map(String::trim)
+                  .map(branch -> branch.replaceAll("(.*)\\/", ""))
+                  .filter(branch -> !branch.isEmpty())
+                  .orElse("test"));
```

Comment:
What we do once we get the ref should be the same in either case - trim, normalize, replace special chars to make it a docker-acceptable format. Also, the current regex looks like it would break on branch names including slashes. How about (suggest you test this, since just writing in a browser):

```suggestion
      return getEnvironmentVariable("CIRCLE_BRANCH")
          .or(getEnvironmentVariable("GITHUB_REF"))
          .map(branch -> branch.replaceAll("refs\\/heads\\/", ""))
          .map(String::trim)
          .map(branch -> branch.replaceAll("[^A-Za-z0-9\\.\\_\\-]", ""))
          .filter(branch -> !branch.isEmpty())
          .orElse("test"));
```